tsconfig.json

```diff
@@ -81,7 +81,7 @@
     // "strictNullChecks": true,                         /* When type checking, take into account 'null' and 'undefined'. */
     // "strictFunctionTypes": true,                      /* When assigning functions, check to ensure parameters and the return values are subtype-compatible. */
     // "strictBindCallApply": true,                      /* Check that the arguments for 'bind', 'call', and 'apply' methods match the original function. */
-    // "strictPropertyInitialization": true,             /* Check for class properties that are declared but not set in the constructor. */
+     "strictPropertyInitialization": false,             /* Check for class properties that are declared but not set in the constructor. */
```

Comment:
Changed this to avoid initializing variables in the constructor of the entities which is redundant